packages/node_modules/@node-red/editor-client/src/js/polyfills.js

```diff
@@ -53,5 +53,31 @@
             Set.prototype.constructor = Set;
         }
 
+        if (window.navigator.msSaveBlob) {
+            // IE does not support data uri scheme for downloading data
+            window.addEventListener("click", function (ev) {
```

Comment:
Adding an event handler on every single click on the page feels like a lot of overhead to fix one button in the UI.

Can we make this a utility function that we call ourselves from the click handler of the download buttons?

Reply:
Hi. Moved download code to a separate function.